### DISCUSSION TITLE
Welcome to lawless-legends Discussions!

### DISCUSSION THREAD

**Discussion by badvision** [badvision/lawless-legends#40] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrewschultz** [comment]
Hey, I didn't realize Github could do this. So it's cool to see. I don't want to go overboard with an introduction, but ... just, it was cool to get a look at Ancient and Lawless Legends both. And if I haven't been following very closely, it's still neat to see what you guys have done.

**Reply by BagoZonde** [reply]
Hey, same here :). The end/almost-end result is outstanding and I'm still impressed with the style and the 3d rendering.
